ExportAs/ExportAs.py

```diff
@@ -54,14 +57,23 @@ def showContextMenuActionsForItem(self, itemID):
     subjectHierarchyNode = pluginHandlerSingleton.subjectHierarchyNode()
     self.exportAsAction.visible = True
     self.exportAsAction.enabled = False
+    self.exportTfmAsAction.visible = True
```

Comment:
To avoid clutter, we may consider hiding the transformed export option if the node is not transformed.

Reply:
Right!